recaf-assembler/src/main/java/me/coley/recaf/assemble/ast/arch/ClassDefinition.java

```diff
@@ -194,6 +204,13 @@ public Module getModule() {
 		return module;
 	}
 
+	/**
+	 * @return Record of the class.
+	 */
+	public Record getRecord() {
```

Comment:
Would be more grammatically correct to say _"Record attributes of the class"_. When this attribute exists, the class _is_ a record. So saying `classThatIsRecord.getRecord()` is odd semantics IMO.